lib/common/imageTool.py

```diff
@@ -42,10 +49,15 @@ def convert_video_to_images(self, input_video_fp, output_image_dir_path, output_
             os.mkdir(output_image_dir_path)
             while result:
                 str_image_fp = os.path.join(output_image_dir_path, "image_%d.jpg" % img_cnt)
-                cv2.imwrite(str_image_fp, image)
+                if (img_cnt >= self.search_range[0] and img_cnt <= self.search_range[1]) or (img_cnt >= self.search_range[2] and img_cnt <= self.search_range[3]):
+                    cv2.imwrite(str_image_fp, image)
+                self.image_list.append({"time_seq": vidcap.get(0), "image_fp": str_image_fp})
                 result, image = vidcap.read()
                 img_cnt += 1
-                self.image_list.append({"time_seq": vidcap.get(0), "image_fp": str_image_fp})
+        if self.search_range[0] < 0: self.search_range[0] = 0
+        if self.search_range[1] > len(self.image_list): self.search_range[3] = len(self.image_list)
+        if self.search_range[2] < 0: self.search_range[0] = 0
+        if self.search_range[3] > len(self.image_list): self.search_range[3] = len(self.image_list)
```

Comment:
the condition of range [1] and [2] have problem. it will correct the wrong var.

```python
        if self.search_range[0] < 0:
            self.search_range[0] = 0
        if self.search_range[1] > len(self.image_list):
            self.search_range[1] = len(self.image_list)
        if self.search_range[2] < 0:
            self.search_range[2] = 0
        if self.search_range[3] > len(self.image_list):
            self.search_range[3] = len(self.image_list)
```
or
```python
        self.search_range[0] = 0 if self.search_range[0] < 0 else self.search_range[0]
        self.search_range[1] = len(self.image_list) if self.search_range[1] > len(self.image_list) else self.search_range[1]
        self.search_range[2] = 0 if self.search_range[2] < 0 else self.search_range[2]
        self.search_range[3] = len(self.image_list) if self.search_range[3] > len(self.image_list) else self.search_range[3]
```